README.md

```diff
@@ -1,64 +1,45 @@
 # OpenFeature Specification (Draft)
 
-[![Roadmap](https://img.shields.io/static/v1?label=Roadmap&message=public&color=green)](https://github.com/orgs/open-feature/projects/1)
-[![Contributing](https://img.shields.io/static/v1?label=Contributing&message=guide&color=blue)](https://github.com/open-feature/.github/blob/main/CONTRIBUTING.md)
-[![Code of Conduct](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](https://github.com/open-feature/.github/blob/main/CODE_OF_CONDUCT.md)
+[![Roadmap](https://img.shields.io/static/v1?label=Roadmap&message=public&color=green)](https://github.com/orgs/open-feature/projects/1) [![Contributing](https://img.shields.io/static/v1?label=Contributing&message=guide&color=blue)](https://github.com/open-feature/.github/blob/main/CONTRIBUTING.md) [![Code of Conduct](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)](https://github.com/open-feature/.github/blob/main/CODE_OF_CONDUCT.md)
 
 This repository describes the requirements and expectations for OpenFeature.
 
-> :warning: Ongoing research can be found in the
-> [research repo](https://github.com/open-feature/research). For definitions of
-> key terminology, see the [glossary](./specification/glossary.md).
+> :warning: Ongoing research can be found in the [research repo](https://github.com/open-feature/research). For definitions of key terminology, see the [glossary](./specification/glossary.md).
 
 ## Design Principles
 
-- Compatibility with existing open source and commercial feature flag offerings
-- Simple, understandable API
-- Vendor agnostic
-- First class Kubernetes support
-- Native support for OpenTelemetry
+- Compatibility with existing feature flag offerings
+- Simple, understandable APIs
+- Vendor agnosticism
+- Language agnosticism
+- Low/no dependency
+- Extensibility
```

Comment:
This is the material change. Please let me know if you agree, or not, or if there's anything we should add.